README.md

```diff
@@ -36,23 +36,27 @@ Examples:
 ```
 bot create milestone "15 Sept 2017" in "datahq/docs"
 bot create milestone all "15 Sept 2017"
-bot close milestone "Sprint - 15 Sept 2017" in "datahq/docs"
+bot close milestone "15 Sept 2017" in "datahq/docs"
 ```
 
 To create an issue on GitHub:
 
 ```
 bot issue "title" about "body" in "myorg/myrepo"
+bot issue "title" in "myorg/myrepo"
 ```
 `title` - the title of the issue.
 `body` - the contents of the issue.
 
 Example:
 ```
-bot issue "This is test issue" about "We want to test our bot create  command" in "datahq/docs"
+bot issue "title of the issue" about "body" in "datahq/docs"
+bot issue "title of the issue" in "datahq/docs"
 ```
 ## Tags
 
+Bot can record channel specified in `config.json` and record into specified Google Spreadsheet, Gist
```

Comment:
I do not get this sentence. Think some commas are missing.

Reply:
@zelima How about `Bot can listen to a specific channel and record it in specified Google Spreadsheet, Gist configured in `config.json`

Reply:
much better. how about 
`Bot can listen to a predefined channels ...`